app/setup.php

```diff
@@ -19,25 +28,20 @@
         wp_enqueue_script('comment-reply');
     }
 
-    $styles = ['styles/app.css'];
-
-    foreach ($styles as $stylesheet) {
-        if (asset($stylesheet)->exists()) {
-            wp_enqueue_style('sage/' . basename($stylesheet, '.css'), asset($stylesheet)->uri(), false, null);
-        }
-    }
```

Comment:
we had this because when running browsersync, the css file isn't there. i haven't tested default sage 10 dev process in a while. so if this isn't needed anymore, then ok. but if we're still not writing CSS file, then we should probably leave this block in there.

Reply:
ah, yeah it's not an issue anymore.